src/inspect_ai/tool/_tools/_web_browser/_web_browser.py

```diff
@@ -97,11 +116,15 @@ def go_without_interactive_docs(tool: Tool) -> Tool:
 # custom viewer for interactive tool calls that shows a truncated
 # version of current the web accessiblity tree if available
 
+WEB_BROWSER_HTML = "web_browser:html"
 WEB_BROWSER_AT = "web_browser:at"
 
 
-def web_at_viewer(call: ToolCall) -> ToolCallView:
-    # get the web accessiblity tree, if we have it create a view from it
+def web_viewer(call: ToolCall) -> ToolCallView:
+    web_html = store().get(WEB_BROWSER_HTML, "")
```

Comment:
Another reason I don't love HTML here is that we end up resenting the entire page, which will get truncated and make it very difficult for human reviewers to see what the model is doing (note below we can vector in on exactly what is being clicked and show a coherent snippet around it). It's quite important to have human oversight for browser tools and accessibility trees seem much better suited to this.